libcodechecker/libhandlers/store.py

```diff
@@ -255,10 +256,7 @@ def consume_plist(item):
         os.chdir(metadata_dict['working_directory'])
 
     buildaction = build_action.BuildAction()
-    if os.path.basename(f).startswith("clangsa_"):
-        buildaction.analyzer_type = analyzer_types.CLANG_SA
-    elif os.path.basename(f).startswith("clang-tidy_"):
-        buildaction.analyzer_type = analyzer_types.CLANG_TIDY
+    buildaction.analyzer_type = analyzer_types.CLANG_SA
```

Comment:
what do we use the buildaction.analyzer_type field in the strore? If not used, remove.